api/src/main/java/org/apache/apex/api/users/ControlTuple.java

```diff
@@ -16,15 +16,15 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.apex.api;
+package org.apache.apex.api.users;
```

Comment:
EOF by definition is a control tuple or a marker. My preference will be not to repeat `ControlTuple` in each control tuple class name. Note that my question was regarding the full name, including the package name where they will be defined. To re-phrase it, will control tuples such as EOF and SOB share the same package and if yes, should they share it with the ControlTuple.